test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.ConnectionLifecycle.cs

```diff
@@ -153,6 +162,7 @@ Task OnTransportStart()
                 using (StartLog(out var loggerFactory))
                 {
                     var startCounter = 0;
+                    var availableTransports = 3;
```

Comment:
Use `IsWebSocketsSupported` here to set the correct number rather than doing it down in the assert.